third_party/s2n-bignum/s2n-bignum-imported/arm/aes/aes-xts-dec-armv8.S

```diff
@@ -0,0 +1,709 @@
+// Auto-added during import for AWS-LC symbol prefixing support
+#include <openssl/boringssl_prefix_symbols_asm.h>
+#include <openssl/asm_base.h>
+
+#ifdef __APPLE__
```

Comment:
Doesn't `#include "_internal_s2n_bignum.h"` have all these definitions already. Can we reuse them?